.github/workflows/test build.yml

```diff
@@ -0,0 +1,25 @@
+name: Test Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      
+      - name: Install dependencies and build
+        working-directory: ./website
+        run: |
+          npm;
+          npm build;
+          npm test
```

Comment:
Add EOF @Ayush7614

Reply:
@Siddhant-K-code sir doesn`t understand what are you say End of Line (EOF) 
npm;
npm build;
npm test.
Like this ?

Reply:
yeah, I am saying that. 
Just add a line below `npm test` only.

So, that it will fix this : 
![image](https://user-images.githubusercontent.com/55068936/128464843-c64b4c3b-6d34-4cb5-8d73-98e8ad0ff5ed.png)